vtex-apps/google-tag-manager
# Feat/ga4 new add shipping info event

#### What is the purpose of this pull request?

This PR intends to add the `add_shipping_info` GA4 event to be sent when `vtex:addShippingInfo` is received.

[`add_shipping_info` event](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?hl=en&client_type=gtag#add_shipping_info)|
-|
<img width="883" alt="Captura de Tela 2023-03-10 às 15 36 56" src="https://user-images.githubusercontent.com/36740164/224398378-5a713d2e-2937-4ab6-b36b-ee3d65a63722.png">



#### How should this be manually tested?

- In your local environment, inside the `google-tag-manager` app repository, go to the branch `feat/ga4-new-add-shipping-info-event`;
- Login in your preferred account and workspace;
- Link the app;
- Go to the Admin and access the App Store. Look for the Google Tag Manager app and access the `Settings`;
- Check the `Merge Universal Analytics and Google Analytics 4 Events` option;
- Access the store and perform add a shipping info on checkout page or in My Account > Addresses;
- Check the `dataLayer` object (just type `dataLayer` in the console and press enter). The event data should be in the list.

#### Types of changes
